addons/sale/models/product_document.py

```diff
@@ -8,15 +8,22 @@ class ProductDocument(models.Model):
 
     attached_on = fields.Selection(
         selection=[
-            ('quotation', "Quotation"),
-            ('sale_order', "Confirmed order"),
+            ('hidden', "Hidden"),
+            ('quotation', "On quote"),
```

Comment:
```suggestion
            ('quotation', "On quotation"),
```
TBC, but from my knowledge, "quote" is quite informal
cc: @dalaOdoo

Reply:
Maybe we should change it everywhere then... =P (Inconsistencies everywhere ATM...)